src/app/useDeepLinks.ts

```diff
@@ -0,0 +1,92 @@
+import dynamicLinks from '@react-native-firebase/dynamic-links'
```

Comment:
these changes are from App.tsx, only i split apart handling initial links / future links to make it easier to read